backend/src/modules/wordfilter/wordfilter.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma';
+import { CreateWordfilterDto } from './create-wordfilter.dto';
+
+@Injectable()
+export class WordfilterService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async createWordfilter(dto: CreateWordfilterDto) {
+    await this.prisma.wordfilter.create({
```

Comment:
dodaj walidowanie regexa, bo jak dodasz błędne wyrażenie np. (() to przy zwracaniu postów będzie wypierdalało 500.
no i jeszcze chyba przydałoby się czyścić board cache po dodaniu regexa, żeby transformowało na nowo stare posty xD

Reply:
a no i może fajniej byłoby trzymać się konwencji w sensie zrobicz jakiś CreateWordfilterResult który będzie miał możliwe pola `OK`, `INVALID_EXPRESSION`, `UNKNOWN_ERROR` tak jak to jest w innych serwisach typu Create